webots_ros2_core/webots_ros2_core/webots_node.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, name, args=None, enableTfPublisher=True):
                 self.get_logger().warn('Impossible to publish transforms because the "supervisor"'
                                        ' field is false.')
 
+        PluginManager(self, plugin_config)
```

Comment:
```suggestion
        self.pluginManager = PluginManager(self, plugin_config)
```

I know we are not using it, but looks cleaner to me (and safer tomato sure that the garbage collector doesn't destroy it).